store/src/java-test/com/zimbra/cs/gal/GalSyncTokenTest.java

```diff
@@ -31,4 +31,20 @@ public void testSyncTokenTimestampZulu() throws ServiceException {
         Assert.assertEquals(fullTimestamp, token.getLdapTimestamp());
         Assert.assertEquals(dateWithHour + "Z", token.getLdapTimestamp("yyyyMMddHHmmss'Z'"));
     }
+
+    @Test
+    public void testSyncTokenMerge() throws ServiceException {
+        String t1 = "20180131045916.000Z:b1010a37-e08d-45d4-b69b-1ea411a75138:11";
+        String t2 = "20180131045916.000Z:b1010a37-e08d-45d4-b69b-1ea411a75138:12";
+        GalSyncToken token1 = new GalSyncToken(t1);
+        GalSyncToken token2 = new GalSyncToken(t2);
+        token2.merge(token1);
+        Assert.assertEquals(t2, token2.toString());
```

Comment:
Please use the variants which have a description as the first argument to make it easier to see from the report what is failing when a test fails.

Reply:
will do.